docs/code_ref/sunpy_stability.yaml

```diff
@@ -5,10 +5,10 @@ data:
   status: mature
   comments: Should see no major changes.
 database:
-  status: dev
-  comments: While it has not changed recently, it could be subject to major revisions in future.
+  status: pendingdep
+  comments: Pending deprecation. See https://community.openastronomy.org/t/deprecating-sunpy-database/495/2 for community discussion.
 image:
-  status: dev
+  status: stable
   comments: While it has not changed recently, it could be subject to major revisions in future.
```

Comment:
If we are happy with what's there should we not change this description?

Reply:
👍 I forgot, will do